aptos-move/aptos-release-builder/data/coin_to_fa_migration.yaml

```diff
@@ -0,0 +1,16 @@
+---
+remote_endpoint: ~
+name: coin_to_fa_migration
+proposals:
+  - name: enable_coin_to_fa_migration
+    metadata:
+      title: "Enable the mapping from coin to fungible asset including APT"
```

Comment:
"Enable the mapping for coin to fungible asset migration, including APT" ?

Reply:
what's wrong?